espnet/nets/pytorch_backend/gtn_ctc.py

```diff
@@ -0,0 +1,84 @@
+import torch
+import gtn
+
+class GTNCTCLossFunction(torch.autograd.Function):
```

Comment:
It looks like this code is mostly copied from https://github.com/facebookresearch/gtn_applications/blob/master/utils.py#L251

I would recommend documenting as such for continuity purposes.

Reply:
Yes - it's documented now.